README.md

```diff
@@ -67,21 +67,18 @@ $ docker run -it --rm --name my-running-app my-nodejs-app
 If you prefer Docker Compose:
 
 ```yml
-version: "2"
 services:
   node:
-    image: "node:8"
+    image: "node:22"
     user: "node"
     working_dir: /home/node/app
     environment:
       - NODE_ENV=production
     volumes:
       - ./:/home/node/app
-    expose:
-      - "8081"
     ports: # use if it is necessary to expose the container to the host machine
-      - "8001:8001"
-    command: "npm start"
+      - "8888:8888"
```

Comment:
One last question: is there any reference information here that can help us confirm whether changing from port 8001 to 8888 is a better, more widely used, and more modern approach? Thanks!

Reply:
Thank you for your explanation. I am not entirely certain about the impact of this adjustment at the moment. If there is a consensus that this revision is objectively better, I am happy to support it. Otherwise, keeping the current version unchanged for now might help us move forward more smoothly. I would appreciate your thoughts.